src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task{
+    private String eventDate;
+    public Event (String description, String eventDate) {
```

Comment:
Perhaps add a space between the method and the variable? It might help with readability, since you've already includes spaces between methods.